packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -785,42 +536,14 @@ void main() {
         ),
     );
     await gesture.up();
-  });
+  }, skip: isBrowser);
```

Comment:
Does the overlay cause issues with browser tests?

Reply:
No, updated.